ui/plugin-system/src/runtime/time-series-queries.ts

```diff
@@ -56,6 +87,7 @@ export function useTimeSeriesQueries(definitions: TimeSeriesQueryDefinition[], o
 
   return useQueries({
     queries: definitions.map((definition) => ({
+      // TODO: Need a way to load multiple plugins at once to call plugin.dependsOn for the queryKey
```

Comment:
@LukeTillman I think we may need a `usePlugins` here so we can load multiple plugins at once. We need to filter the variableState to only the variables that a query uses to get a queryKey but this is an async operation.